doc/format_specification.rst

```diff
@@ -0,0 +1,179 @@
+
+For a more detailed description of the parsing of mathematical expressions (e.g. for  `formula`) we refer to the [corresponding section](#parsing-of-mathematical-equations) of this documentation.
+
+
+
+observables \[optional\]
+------------------------
+
+.. code-block:: yaml
+
+  observables:
+
+      - observableId: Obs_1
+        observableFormula: x_1 + x_2
+
+        noiseFormula: noiseParameter1
+        noiseDistribution: normal
+
+      - observableId: Obs_2
+        ...
+
+Defines the **observables**. Observables are not part of the SBML standard. If the SBML is generated via the `yaml2sbml.yaml2sbml` command and the `observables_as_assignments` flag is set to `True`, observables are represented as assignments to parameters of the form observable_<observable_id>.
+If the SBML is created via `yaml2sbml.yaml2petab`, observables are represented in the PEtab observables table. The entries are written to the corresponding columns of the PEtab observable table. According to the PEtab standard, an observable table can take the following entries:  `observableId, observableName, observableFormula, observableTransformation, noiseFormula, noiseDistribution`.
+
+For a detailed discussion see the `corresponding part of the PEtab documentation <https://github.com/PEtab-dev/PEtab/blob/master/doc/documentation_data_format.rst#observables-table>`_.
+
+
+
+conditons \[optional\]
+----------------------
+
+.. code-block:: yaml
+
+  conditions:
+
+      - conditionId: condition1
+        p_1: 1
+        x_1: 2
+        ...
+
+
+Conditions allows to set parameters or initial conditions of states to a numeric value/unknown parameter. This allows to specify different experimental setups in the data generation (e.g. different initial conditions for different runs of an experiment). 
+
+The "trivial condition table" (if only one setup exist) is generated by:
+
+.. code-block:: yaml
+
+  conditions:
+        - conditionId: condition1
+
+
+For a detailed discussion see the `corresponding part of the PEtab documentation <https://github.com/PEtab-dev/PEtab/blob/master/doc/documentation_data_format.rst#condition-table>`_.
+
+
+
+Parsing of mathematical equations
+---------------------------------
+
+ Throughout `yaml2sbml` formulas are parsed by `libsbml's` `parseL3Formula` function. Further information on the syntax are given by:
+
+*  the `working with math <http://sbml.org/Special/Software/libSBML/docs/formatted/python-api/libsbml-math.html>`_ - section of the `libsbml` documentation.
+*  the `documentation <http://sbml.org/Special/Software/libSBML/docs/formatted/python-api/namespacelibsbml.html#ae79acc3be958963c55f1d03944add36b>`_ of `libsbml.parseL3Formula`.
+
+
+This gives access to e.g.:
+
+*  +, -, *, / and power,
+*  trigonometric/hyperbolic functions, 
+*  exponential/logarithmic functions,
+*  piecewise defined functions,
+*  boolean expressions like "<".
```

Comment:
```suggestion
*  `+`, `-`, `*`, `/`, and exponentiation;
*  trigonometric/hyperbolic functions;
*  exponential/logarithmic functions;
*  piecewise defined functions; and
*  boolean expressions like "a lesser than b" (`a < b`).
```
Semi-colons since one of the list items contains commas.
Backticks because otherwise the `*` appeared as a link :thinking: